resumes/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .views      import *
+
+urlpatterns = [
+    path("/resume", ResumeView.as_view()),

```

Comment:
`:8000/resumes/resume`가 의도하신 엔드포인트 주소가 맞나요?
```suggestion
    path("", ResumeView.as_view()),

```